testing/src/test/java/org/oppia/android/testing/FakeFirebaseAuthWrapperImplTest.kt

```diff
@@ -61,14 +61,14 @@ class FakeFirebaseAuthWrapperImplTest {
     fakeFirebaseAuthWrapperImpl.simulateSignInSuccess()
 
     val authState = fakeFirebaseAuthWrapperImpl.getAuthState()
-    assertThat(authState).isInstanceOf(FakeAuthState.Success::class.java)
+    assertThat(authState).isInstanceOf(FakeAuthState.SUCCESS::class.java)
```

Comment:
Nit: can just do ``isEqualTo`` for the enum constant. Ditto everywhere else in the PR.